documents/meetings/conference_call.md

```diff
@@ -1,12 +1,29 @@
 ## Conference Call
 
 1. What are the most important aspects to include in the project, and discuss in the presentation?
+  - which tools we are using *He provided some suggestions such as React, Jest/Selenium, istanbul/visual studio*
+  - Show the division of server side vs Javascript and give some reasoning for it. *(I’m not sure this is relevant anymore)*
 2. Is there a target demographic for the project?
+
 3. Is the project data private? Public? Secured?
+  - need credentials to log in possibly using google authentication or .netcore
+  - will need security testing as security is a priority
 4. Is there a specific platform for the application?
+
 5. Portability?
+  - cross integration is medium priority
+  - tablet and laptop compatible is preferred
+  - phone screens are low priority
 6. What kind of data is to be presented?
+  - tabluar in regards to time with either just starting or starting and ending times
+  - He will send us the data sets that he has in mind
 7. The specifications for the presentation mention 'Wiki Pages'. What does it mean? And how do we utilize them?
+  - contains all relevant information for example:
+    - this is how you set up ____
+    - this document is *here* in the repo
 8. What needs to be discussed for the smoke-tests? any tools? Specifics? Tenses?
+
 9. How big are the data sets?
+  - *Not that big? I think this is were Mesa could clarify*
```

Comment:
I think Mesa clarified this on slack

Reply:
This was clarified in [this](https://github.com/UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2/blob/master/documents/meetings/stakeholder10.4.md) meeting